lib/ansible/modules/windows/win_service.ps1

```diff
@@ -88,6 +149,14 @@ Function Get-ServiceInfo($name) {
         $description = ""
     }
 
+    $recovery = Get-RecoveryActions($svc.Name)
+
+    if ('' -ne $recovery) {
+        $recovery = RecoveryActionsToHuman($recovery)
+    } else {
+        $recovery = @{}
+    }
+
```

Comment:
Why not simply return `$null` or @{} in teh function?

Reply:
Because I use that function elsewhere and I don't want the conversion to "Human"